src/invert/laplace/invert_laplace.cxx

```diff
@@ -704,6 +704,34 @@ void Laplacian::tridagMatrix(dcomplex *avec, dcomplex *bvec, dcomplex *cvec,
   }
 }
 
+void Laplacian::calcDelp2Coefs() {
+  TRACE("Laplacian::calcDelp2Coefs");
+
+  static bool initializedDelp2Coefs = false ;
```

Comment:
I think this shadows the class level variable of the same name. I'd guess you want one or the other.